kolena/io.py

```diff
@@ -24,8 +24,10 @@
 
 
 def _serialize_dataobject(x: Any) -> Any:
-    if isinstance(x, list):
-        return [item._to_dict() if isinstance(item, DataObject) else item for item in x]
+    if isinstance(x, (list, tuple)):
```

Comment:
seems like Union is only supported [after 3.10](https://docs.python.org/3/library/functions.html#isinstance)

Reply:
oh, I will use `or` instead, thanks for pointing this out